docs/dtds/checkstyle-metadata_1_0.dtd

```diff
@@ -7,145 +7,145 @@
     "http://eclipse-cs.sourceforge.net/dtds/checkstyle-metadata_1_0.dtd">
```

Comment:
@Calixte When we discussed `docs` at https://github.com/checkstyle/eclipse-cs/pull/457#discussion_r1091318388 , it was my understanding that `docs` was not updated real-time with the snapshots and is only updated during a release. Did I misunderstand something?

CI at https://github.com/checkstyle/eclipse-cs/pull/460 didn't run travis, so this is why it was missed.
CI at https://github.com/checkstyle/eclipse-cs/pull/457 did run travis and it reported no issues, but `docs` was not required to be updated.

How come my PR did not updated `docs` but yours did? Is it related to the change you made in [net.sf.eclipsecs.doc/build.properties](https://github.com/checkstyle/eclipse-cs/pull/460/files#diff-c1a23c35875bfc9d7e2032a9e48486c40759e09ec9e01fe19898b9da04f9b835) ?

Reply:
I assume so, yes. The docs directory will get some changes from time to time, when maven runs. I just never commit them, because the release process does it.